src/dodal/devices/focusing_mirror.py

```diff
@@ -107,30 +114,49 @@ def voltage_channels(self) -> list[MirrorVoltageDevice]:
         ]
 
 
-class FocusingMirror(Device):
+class FocusingMirror(StandardReadable):
     """Focusing Mirror"""
 
-    def __init__(self, bragg_to_lat_lut_path, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(
+        self, name, prefix, bragg_to_lat_lut_path=None, x_y_sufffixes=("X", "Y")
+    ):
         self.bragg_to_lat_lookup_table_path = bragg_to_lat_lut_path
+        self.yaw_mrad = Motor(prefix + "YAW")
+        self.pitch_mrad = Motor(prefix + "PITCH")
+        self.roll_mrad = Motor(prefix + "ROLL")
+        self.x_mm = Motor(prefix + x_y_sufffixes[0])
```

Comment:
Maybe one too many f's

```suggestion
        self.x_mm = Motor(prefix + x_y_suffixes[0])
```